src/common/style/vanilla/breadcrumbs.scss

```diff
@@ -1,3 +1,4 @@
+@import 'vanilla-framework/scss/_utilities_clearfix.scss';
```

Comment:
Breadcrumb pattern was changed and now relies on `%clearfix` but doesn't import it on its own.